gg/render.go

```diff
@@ -325,14 +326,14 @@ func renderScale(svg *svg.SVG, dir rune, scale Scaler, ticks plotEltTicks, pos i
 			}
 			have[p] = true
 			if dir == 'x' {
-				path = append(path, fmt.Sprintf("M%.6g %dv%.6g", p, pos, -t.length))
+				fmt.Fprintf(&path, "M%.6g %dv%.6g", p, pos, -t.length)
 			} else {
-				path = append(path, fmt.Sprintf("M%d %.6gh%.6g", pos, p, t.length))
+				fmt.Fprintf(&path, "M%d %.6gh%.6g", pos, p, t.length)
 			}
 		}
 
-		svg.Path(strings.Join(path, ""), "stroke:#888; stroke-width:2") // TODO: Theme
 	}
+	svg.Path(path.String(), "stroke:#888; stroke-width:2") // TODO: Theme
```

Comment:
The SVG spec recommends adding newlines every <255 characters (but the newline must be at the boundary of a command); are we worried that we'll now produce paths that are too long? (Of course, the paths for the plot itself are probably even worse...)